docs/api/effector-react/createComponent.md

```diff
@@ -43,9 +43,11 @@ const MyCounter = createComponent($counter, (props, state) => (
 ))
 
 const MyOwnComponent = () => {
-  // any stuff here
+  // any staff here
```

Comment:
**question**: looks like older option was more correct 🤔

Reply:
@AlexandrHoroshih I noticed this change too, but I didn't make it.)))